### PR TITLE
FLUT-6089 - [Others] Resolved the concerns in UG improvements

### PR DESCRIPTION
**Pull request by PavithraNandhiniRamachandran**
## Description ##

Resolved the concerns in UG improvements

### CODE REVIEW

File: Flutter/cartesian-charts/chart-types/histogram-chart.md [hunk 1]
```diff
@@ -31,125 +31,129 @@ You can collapse the normal distribution curve using the [`showNormalDistributio
...
-                yValueMapper: (ChartData sales, _) => sales.y)]))));
-        }
+        @override
+        Widget build(BuildContext context) {
+        return Scaffold(
+            body: Center(
+                child: Container(
+                    child: SfCartesianChart(series: <ChartSeries>[
+                    HistogramSeries<ChartData, double>(
+                    dataSource: <ChartData>[
```
Comment (dharanidharandharmasivam) [inline Flutter/cartesian-charts/chart-types/histogram-chart.md:41]: move the data to new file

**Review by dharanidharandharmasivam — CHANGES_REQUESTED** [review]

File: Flutter/cartesian-charts/axis-customization.md [hunk 2]
```diff
@@ -988,10 +990,10 @@ By default, the chart is rendered with primary x axis and primary y axis. But, t
     }
 
     class ChartData {
-        ChartData(this.year, this.sales, [this.numeric]);
-        final String year;
-        final double? sales;
-        final double? numeric;
+        ChartData(this.x, this.y, this.numeric);
```
[the remark this answers is not in the document]
Comment (PavithraNandhiniRamachandran) [reply Flutter/cartesian-charts/axis-customization.md:993]: have changed to y

File: Flutter/cartesian-charts/chart-types/histogram-chart.md (hunk 1 above)
Comment (PavithraNandhiniRamachandran) [reply Flutter/cartesian-charts/chart-types/histogram-chart.md:41]: moved the data to new file

**Review by dharanidharandharmasivam — APPROVED** [review]

**Comment by dharanidharandharmasivam** [comment]
Hi Team,

The changes are fine and thus approved.

Thanks,
Dharani.

**Pull request merged**